wiki/Beatmap_Editor/Song_Setup/en.md

```diff
@@ -1,222 +1,273 @@
+[beatmap_CS_1]: https://osu.ppy.sh/beatmapsets/2008 "Trauma Center: New Blood - Opening by Larto"
+[beatmap_CS_2]: https://osu.ppy.sh/beatmapsets/1208 "ORANGE RANGE - Ikenai Taiyou by Echo"
+[beatmap_PS]: https://osu.ppy.sh/beatmapsets/41820 "Beautiful Day - Bang! Bang! Bang! by -SiN-"
+
+<!-- img -->
+[SS_nav]: ./img/Song_Setup_nav.jpg "Where the Song Setup is located in the Beatmap Editor"
+[SS_gen]: ./img/SS_General.jpg "An example of complete metadata (General tab) in Song Setup"
+[SS_audio]: ./img/SS_Audio.jpg "Audio section. The right section will be shown if the beatmap contains custom timing on volume settings"
+[SS_diff]: ./img/SS_Difficulty.jpg "Difficulty section. The right section is for osu!mania-specific beatmap."
+[SS_colours]: ./img/SS_Colours.jpg "Colours section"
+[SS_design]: ./img/SS_Design.jpg "Design section"
+[CS]: ./img/Circle_sizes.png "Overview of various Circle Sizes"
+[AR]: /wiki/shared/ARTable.jpg "Detailed AR chart"
+[OD]: ./img/ODTable.jpg "Detailed OD chart"
+[SS_ps]: ./img/SS_Skin_eg.jpg "The preferred skin is set as grungosu and there is no skin elements in the beatmapset folder as shown in the top-half. If the player have the skin name in the Skins folder, that skin will be used as shown in the bottom-half."
+[SS_adv]: ./img/SS_Advanced.jpg "Advanced section"
+[SS_sl]: ./img/SS_SL_eg.jpg "Stack Leniency"
+
+# Song Setup
+
+![Where the Song Setup is located in the Beatmap Editor][SS_nav]
+
+Song Setup is an important screen in [beatmapping][Beatmapping]. It contains metadata and general beatmap configuration settings for the beatmap.
+
+## General
 
 ### Song and Map Meta Data
 
-![An example of complete data (General tab)](img/Song_Setup.jpg "An example of complete data (General tab)")
+![An example of complete metadata (General tab)][SS_gen]
 
-[Guide on finding Metadata](https://osu.ppy.sh/forum/t/249288).
+_See also: [\[Guide\] Metadata][forum_metadata]_
 
-Not every song needs something in the "Source" field. The source is only for songs that are "from" something, such as a video game theme song, or something from the soundtrack of an anime. Regular rock and pop songs can just leave the source field blank.
+Title|Description
+:---|:---
+Artist | **The band, singer, composer, or group that performed or created the song.** It will be pulled automatically from the `.mp3` file's metadata but please double-check to ensure that it is spelled and punctuated correctly. **Anime/Game titles, Vocaloid names, or companies do not belong here (they belong to Tags).** The composer for anime and game songs can be found on [Wikipedia](https://www.wikipedia.org "Wikipedia - The Free Encyclopedia") or other reliable sources such as [AnimeNewsNetwork](https://www.animenewsnetwork.com/ "AnimeNewsNetwork") or [MyAnimeList](https://myanimelist.net/ "MyAnimeList"). Foreign names should be presented in the same name order as used on Wikipedia (generally, Given Name followed by Family Name; the western order, for consistency).
```

Comment:
Rewrite to:-

```
It will be pulled automatically from the `.mp3` file's metadata but please double-check to ensure proper spelling and punctuation used.
```

Ref: https://www.dailywritingtips.com/comma-before-but/ and https://english.stackexchange.com/a/28281